components/contributors/InfoCard.tsx

```diff
@@ -5,16 +5,30 @@ import Image from "next/image";
 import Link from "next/link";
 import { BsSlack } from "react-icons/bs";
 import { env } from "@/env.mjs";
+import { getLeaderboardData } from "@/app/api/leaderboard/functions";
+import { parseDateRangeSearchParam } from "@/lib/utils";
 
-export default function InfoCard({
+export default async function InfoCard({
   contributor,
-  minimal = true,
+  isRank,
+  // minimal = true,
   isClickable = false,
 }: {
   contributor: Contributor;
-  minimal?: boolean;
+  isRank?: boolean;
```

Comment:
Repeating... rename it to `showRank` as `isRank` isn't an appropriate name for what this prop does.

```suggestion
  showRank?: boolean;
```